TestKit/Sources/TestKit/XCTestCase+Wait.swift

```diff
@@ -14,10 +14,12 @@ extension XCTestCase {
     /// }
     /// ```
     ///
-    func waitForExpectation(description: String? = nil,
-                            timeout: TimeInterval = Constants.expectationTimeout,
+    public func waitForExpectation(description: String? = nil,
+                            count: Int = 1,
+                            timeout: TimeInterval = 5.0,
```

Comment:
nit: indentation from `public` 🙂

Reply:
Thanks! Fixed in 6281cb611ec64e9ed77652769964b901136c4841.